src/components/playback-controls/playback-controls.tsx

```diff
@@ -60,9 +61,10 @@ class PlaybackControls extends Component<any, any> {
     if (shouldUpdate) {
       this.setState({shouldUpdate: false});
     }
+    const imageClass = this.props.image ? 'playkit-image' : '';
```

Comment:
isn't the className image and not playkit-image ?
also its more correct to use style.[className], because the actual class name will have a hash added to it